spec/models/user_spec.rb

```diff
@@ -117,6 +133,37 @@
         expect(User.not_in_experience).to contain_exactly(user_not_in_experience)
       end
     end
+
+    describe '#by_roles_in' do
```

Comment:
this is a scope wich is a class method, so the correct notation is `.by_roles_in` instead `#by_roles_in`

Reply:
May I change all the other scope methods in this PR too?

Reply:
yes pls